src/libsyntax/parse/lexer/mod.rs

```diff
@@ -1873,13 +1885,15 @@ fn is_block_doc_comment(s: &str) -> bool {
     res
 }
 
+/// Determine wether `c` is a valid start for an ident. `include_numeric` is used for error
```

Comment:
There doesn't seem to be an `include_numeric`.

Reply:
nit: ‘wether’ -> ‘whether’